eox_azure/storage.py

```diff
@@ -22,59 +19,8 @@ def __init__(self, container=None, url_expiry_secs=None, *args, **kwargs):  # py
         """
 
         super(AzureStorageExtended, self).__init__()
-        self._connection = None
 
         self.url_expiry_secs = url_expiry_secs
```

Comment:
It looks like in django-storages==1.8.0 the name of this config is 
self.expiration_secs